<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="APIAutomation" duration-ms="4647" started-at="2020-10-09T09:57:01Z" finished-at="2020-10-09T09:57:06Z">
    <groups>
    </groups>
    <test name="APIRestAssuredAutomation" duration-ms="4647" started-at="2020-10-09T09:57:01Z" finished-at="2020-10-09T09:57:06Z">
      <class name="testCases.PDPAPI">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:testCases.PDPAPI@42dafa95]" name="beforeClass" is-config="true" duration-ms="14" started-at="2020-10-09T09:57:01Z" finished-at="2020-10-09T09:57:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="getPDP()[pri:0, instance:testCases.PDPAPI@42dafa95]" name="getPDP" duration-ms="2620" started-at="2020-10-09T09:57:01Z" finished-at="2020-10-09T09:57:04Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [417]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [417]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at apiVerifications.APIVerification.responseCodeValidation(APIVerification.java:17)
at testCases.PDPAPI.getPDP(PDPAPI.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPDP -->
      </class> <!-- testCases.PDPAPI -->
      <class name="testCases.GetCartAPI">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:testCases.GetCartAPI@6500df86]" name="beforeClass" is-config="true" duration-ms="1" started-at="2020-10-09T09:57:04Z" finished-at="2020-10-09T09:57:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="getCart()[pri:0, instance:testCases.GetCartAPI@6500df86]" name="getCart" duration-ms="1579" started-at="2020-10-09T09:57:04Z" finished-at="2020-10-09T09:57:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getCart -->
      </class> <!-- testCases.GetCartAPI -->
      <class name="testCases.ClearCartAPI">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:testCases.ClearCartAPI@402a079c]" name="beforeClass" is-config="true" duration-ms="0" started-at="2020-10-09T09:57:05Z" finished-at="2020-10-09T09:57:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="clearCart()[pri:0, instance:testCases.ClearCartAPI@402a079c]" name="clearCart" duration-ms="205" started-at="2020-10-09T09:57:05Z" finished-at="2020-10-09T09:57:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearCart -->
      </class> <!-- testCases.ClearCartAPI -->
    </test> <!-- APIRestAssuredAutomation -->
  </suite> <!-- APIAutomation -->
</testng-results>
